moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_V
 ament_target_dependencies(${MOVEIT_LIB_NAME} SYSTEM
   BULLET
 )
+if(APPLE)
+  target_link_directories(${MOVEIT_LIB_NAME} PUBLIC ${BULLET_LIBRARY_DIRS})
```

Comment:
`ament_target_dependencies` should already search for the correct library dirs. (see [here](https://github.com/ament/ament_cmake/blob/master/ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake#L116)). Or is it actually missing the `PUBLIC` keyword. I didn't check thoroughly, but Bullet doesn't seem to be part of the public API.

Reply:
I believe this is an issue with the custom FindBULLET. Removing it fixed this issue. Let me know if that's okay -- it is quite old, and I think if the CI passes, it should be.